selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -135,14 +135,16 @@ void TogglesPanel::updateToggles() {
                                           "<h4>%6</h4><br>"
                                           "%7")
                                   .arg(tr("openpilot defaults to driving in <b>chill mode</b>. Experimental mode enables <b>alpha-level features</b> that aren't ready for chill mode. Experimental features are listed below:"))
-                                  .arg(tr("🌮 End-to-End Longitudinal Control 🌮"))
+                                  .arg(tr("End-to-End Longitudinal Control" ))
                                   .arg(tr("Let the driving model control the gas and brakes. openpilot will drive as it thinks a human would, including stopping for red lights and stop signs. "
                                        "Since the driving model decides the speed to drive, the set speed will only act as an upper bound. This is an alpha quality feature; mistakes should be expected."))
                                   .arg(tr("Navigate on openpilot"))
-                                  .arg(tr("When navigation has a destination, openpilot will input the map information into the model. This generally improves behavior and allows openpilot to keep left or right appropriately at forks/exits and take turns. "
-                                          "Lane change behavior is unchanged and still activated by the driver. This is an alpha quality feature; mistakes should be expected."))
+                                  .arg(tr("When navigation has a destination, openpilot will input the map information into the model. This generally improves behavior and allows openpilot to keep left or right appropriately at forks/exits. "
```

Comment:
I think "generally improves behavior" can be confusing here; maybe something like "the map input provides context for and informs all driving behavior"